### PR TITLE
fix: web setRem error in cmd bundle

### PR DESCRIPTION
**Pull request by yacheng**
*Before* submitting a pull request, please make sure the following is done...

1. Fork the repo and create your branch from `master`.
2. If you've added code that should be tested, add tests!
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes (`npm test`).
5. Make sure your code lints (`npm run lint`) - we've done our best to make sure these rules match our internal linting guidelines.

### CODE REVIEW

File: packages/universal-transition/src/index.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import {isWeex, isWeb} from 'universal-env';
-import {convertUnit} from 'style-unit';
+import { convertUnit, setRem } from 'style-unit';
+
+if (isWeb) {
```
Comment (yuanyan) [inline packages/universal-transition/src/index.js]: Why setRem() here?

Comment (yacheng) [reply packages/universal-transition/src/index.js]: 部分业务场景会独立打包 style-unit 进模块（主要是 web 中的业务模块打包机制造成的），与 driver 中的不是同一个实例，导致模块中 style-unit 的 defaultRemUnit 始终是 1

File: packages/style-unit/src/index.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ const SUFFIX = 'rem';
 const REM_REG = /[-+]?\d*\.?\d+rem/g;
 
 // Default 1 rem to 1 px
-let defaultRemUnit = 1;
+window._defaultRemUnit = 1;
```
Comment (yuanyan) [inline packages/style-unit/src/index.js]: 这样写每次初始化不是被重置了吗？

Comment (yuanyan) [reply packages/style-unit/src/index.js]: 同时还需考虑下Node环境

File: packages/style-unit/src/index.js [hunk 3]
```diff
@@ -36,7 +36,18 @@ const SUFFIX = 'rem';
 const REM_REG = /[-+]?\d*\.?\d+rem/g;
 
 // Default 1 rem to 1 px
-let defaultRemUnit = 1;
+let defaultRemUnit;
+if (typeof window) {
```
Comment (yuanyan) [inline packages/style-unit/src/index.js]: 这个一直是true哈，整体代码写有些复杂了

**Pull request merged**